### DISCUSSION TITLE
Deliberate support for offline-only mode

### DISCUSSION THREAD

**Discussion by shanberg** [athensresearch/athens#1025] [General]
Some users may only use Athens offline. Some may use it online when possible, but mostly offline. Certain changes may improve the experience for them, like not trying to load remote content (like videos in the Merge from Roam dialog).

What other ways can Athens account for this use case and improve their experience?

**Comment by Limezy** [comment]
As I wrote it in the iOS thread, another use case of offline mode I could think of is the "Airplane mode" when mobile apps will be out

**Comment by seekanddefine** [comment]
Every single user of Athens should be considered a possible offline user. 19 million Americans lack access to fixed-broadband at threshold speeds. In rural areas 1/4-14.5 million people lack access. Worldwide there are 3.7 million people with no internet access at all.  https://www.fcc.gov/reports-research/reports/broadband-progress-reports/eighth-broadband-progress-report

At a minimum baseline support for offline Athens is core to the Athens experience. Full adoption of a tool partially relies on the seamlessness of the adoption transition. Running offline is something that markedly sets Athens apart from its competitors. One of the reasons I adopted Athens as my own tool is `*because*` it works offline.

As a note-taking tool I use daily, lack of support has been an issue in the speed I've adopted the tool because I've experienced questions that demanded I go online to find an answer since day one. My morning routine includes a brain-dump style "Morning pages" that I appreciate typing because it keeps up with the faster thought flow better than writing with a pen. As someone easily distracted, I intentionally don't connect to the internet until this task is finished and my task management goals are delineated. 

This is a big deal in the writing market where people will pay more money to have a simple tool that blocks distractions and works offline in order to work more efficiently. I would argue that Word is in part still more popular as a tool than Google Docs because of it's ability to more seamlessly work offline as well as it's home being located outside of a potentially very distracting browser. When it comes down to it, word count goals are word count goals and they have to be met. Offline access and offline support go hand-in-hand for user experience. 

If our market is researchers that includes writers and Scrivener is considered the go-to for anything long-form. Why? Because it helps you research, compose, and reorganize. Sound familiar? It has no web app, it has no collaboration features and doesn't run at all in Linux! It's difficult to learn because it's rather unintuitive and it's extraneous features are arguably more distracting than not.  So why do people use it? It tracks word counts and other data, has lots of templates and includes a WYSIWYG editor/support markdown which is gold because of ease of use. Let's not forget outliner mode (bullets WOW!) and split screen! Sound like Athens yet? This argument is not for how similar Athens is to Scrivener, while there is some feature matching, they are different tools in many ways. 

I wholeheartedly endorse deliberate support for offline only mode and argue it is something that should be advertised explicitly when we have it. As Athens is THE open-source option, it should back that up with support for users who cannot afford or do not have access to consistent internet as well. 

The day that access and connectivity are no longer problems is still far away. Living in a rural town means I am never surprised if the internet is unexpectedly out. Even when that day comes with connections speeds whirring, I will still intentionally start my day without it until of course I can't remember the shortcut key for deleting to the end of a block or collapsing subordinate blocks or toggling the block preview tooltip. Oh right, tooltips!
